packages/node/src/integrations/localvariables/localvariables-async.ts

```diff
@@ -0,0 +1,287 @@
+import type { Event, EventProcessor, Exception, Hub, Integration, StackFrame, StackParser } from '@sentry/types';
+import { LRUMap, logger } from '@sentry/utils';
+import type { Debugger, InspectorNotification, Runtime } from 'inspector';
+import type { NodeClient } from '../../client';
+
+import type { NodeClientOptions } from '../../types';
+import type { FrameVariables, Options, PausedExceptionEvent, RateLimitIncrement, Variables } from './common';
+import { createRateLimiter, functionNamesMatch, hashFrames, hashFromStack } from './common';
+
+type Session = {
```

Comment:
In other places I see that we use `interface`, is there a specific reason to why we are using `type` in here?

Reply:
I think the general rule is to prefer type over interface unless you're describing a class interface. 

I'm using a type here because you can't define a constructor on an interface!

Reply:
Nice! Thanks.